package.json

```diff
@@ -24,9 +24,13 @@
     "format": "yarn prettier:fix && yarn lint",
     "mocks": "yarn run wiremock --verbose --port 8090 --root-dir test/mocks",
     "test": "node_modules/.bin/jest --testPathPattern=app/bav --coverage",
+    "test:browser": "wait-on tcp:127.0.0.1:8090 tcp:127.0.0.1:5030 && API_BASE_URL=http://127.0.0.1:8090 cucumber-js test/browser/features/**/*.feature --require test/browser/support/env.js  --require test/browser/support/setup.js --require test/browser/step_definitions/**/*.js -f json:./test/reports/cucumber_report.json --tags @success",
+    "test:browser:only": "wait-on tcp:8090 tcp:5030 && cucumber-js test/browser/features/**/*.feature --require test/browser/support/env.js  --require test/browser/support/setup.js --require test/browser/step_definitions/**/*.js --tags @only",
+    "test:browser:ci": "IPV_BASE_URL=http://localhost:5030 npm-run-all -p -r start:ci mocks test:browser && npm run test:browser:report",
     "test:browser:report": "node ./test/utils/multiReportGenerator.js",
-    "test:browser": "wait-on tcp:127.0.0.1:8090 tcp:127.0.0.1:5040 && API_BASE_URL=http://127.0.0.1:8090 cucumber-js test/browser/features/**/*.feature --require test/browser/support/env.js  --require test/browser/support/setup.js --require test/browser/step_definitions/**/*.js -f json:./test/reports/cucumber_report.json",
-    "test:browser:ci": "npm-run-all -p -r start:ci mocks test:browser && npm run test:browser:report",
+    "test:e2e:cd:cucumber": "wait-on tcp:8090 && cucumber-js test/browser/features/**/*.feature --require test/browser/support/env.js  --require test/browser/support/setup.js --require test/browser/step_definitions/**/*.js -f json:./test/reports/cucumber_report.json --tags @success",
+    "test:e2e:cd": "npm-run-all -p -r mocks test:e2e:cd:cucumber && yarn run test:browser:report",
+    "test:e2e": "cucumber-js test/browser/features/**/*.feature --require test/browser/support/env.js  --require test/browser/support/setup.js --require test/browser/step_definitions/**/*.js -f json:./test/reports/cucumber_report.json --tags @e2e",
```

Comment:
Would you be able to tell me/document the difference between all of these commands? I only ever use `browser:test:ci` not sure if I'm doing it right

Reply:
I've tidied this up. 

I've been using browser:test:ci myself for local tests, and test:e2e:cd is being used by the run-tests.sh script